services/ui-src/src/components/export/ExportedEntityDetailsOverlaySection.tsx

```diff
@@ -112,27 +112,27 @@ export function getEntityTableComponents(
       report_planName,
     } = getEntityDetailsMLR(entity);
 
+    const entityHeading = `${idx + 1}. ${
+      section.verbiage.intro.subsection
+    } for:`;
+
     const programInfo = [
       report_programName,
       mlrEligibilityGroup,
       reportingPeriod,
       report_planName,
     ];
 
-    const entityHeading = `${idx + 1}. ${
-      section.verbiage.intro.subsection
-    } for:`;
+    const formatProgramInfo = (index: number, field: string) => {
+      return <p key={index}>{field}</p>;
+    };
 
     return (
       <Box key={uuid()}>
         <Box sx={sx.entityInformation}>
           <Heading sx={sx.entityHeading} as="h3" fontSize={"xl"}>
             {entityHeading}
-            <ul>
-              {programInfo.map((field, index) => (
-                <li key={index}>{renderHtml(field)}</li>
-              ))}
-            </ul>
+            {programInfo.map((field, index) => formatProgramInfo(index, field))}
```

Comment:
Question: Are these fields are coming from user input, from the report template, or from verbiage files?

I think they're user input, but I'm only 80% sure. And if so, then I totally approve this change.

Reply:
Correct, these are indeed user input!